server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/ConsistencyTasksIntegrationTest.java

```diff
@@ -312,7 +334,53 @@ void shouldRecomputeQuotas(GuiceJamesServer server) throws Exception {
     }
 
     @Test
-    void solveCassandraMappingInconsistencyShouldSolveNotingWhenNoInconsistencies() {
+    void shouldSolveMessagesInconsistency(GuiceJamesServer server) {
+        // schema version 6 or higher required to run solve message inconsistencies task
+        String upgradeTaskId = with().post(UPGRADE_TO_LATEST_VERSION)
```

Comment:
Can we give it better name to remove the comment

Reply:
And this piece of code are duplicating in this class, can we solve it?

Reply:
We tend to avoid method extraction in testsas it hurts readability

Reply:
> We tend to avoid method extraction in testsas it hurts readability

I agree with not to method extraction in the test.

I propose:
`void shouldSolveMessagesInconsistencyWithLatestVersion(GuiceJamesServer server)`
and maybe add a case `void shouldNotSolveMessagesInconsistencyWithVersion5OrLower(GuiceJamesServer server)`
And we dont need to put the comment.

Reply:
> and maybe add a case void  
shouldNotSolveMessagesInconsistencyWithVersion5OrLower(GuiceJamesServer server)
> And we dont need to put the comment.

No we won't add this test case as it is already covered at the service level.

We should avoid duplicating tests across layers. Here we should provide a smoke test of the actual test suite, written at the service level.